.github/workflows/ci.yml

```diff
@@ -255,7 +255,7 @@ jobs:
     - uses: actions/checkout@v4
     - uses: actions/setup-python@v5
       with:
-        python-version: '3.8'
+        python-version: '3.10'
```

Comment:
:heart: 

BTW 3.13 will be released in like 6 weeks ;)

Reply:
Then its time to move to 3.12 :D. Sadly seems that moving to 3.10 does not help :S

Reply:
@roigcarlo can you give me a quick rundown on what's going on? I know this oot issue has been around for quite a while, but not much else. Why would changing the python exec help?

Reply:
Basically some python tests fail in windows since July. It began for no apparent reason. I tracked down that there were some updates to the Windows container but that's it. To my knowledge all tests failing involve reading (or maybe just using)  csr matrices at some point.

After much testing I suspect that the error happens while calling the `mm_read_banner` from the matrixmarket lib (`mmio.h` and `mmio.c`).

The problem is that in 3 months I have been unable to reproduce this anywhere, and the windows container used to run in the CI is not public because of licences issues.